### ISSUE TITLE
Login with device doesn't work, PUT response status 422 - Unprocessable entity

### ISSUE THREAD

**Issue by Majestic7979** [dani-garcia/vaultwarden#4031]
I am running behind traefik 2, I get the websockets connection just fine. I also get push notifications. I get the fingerprint phrase popup on my cellphone and in my desktop application.

Doing F12 on the browser and the Bitwarden desktop to see what's going on...

GET status 200 response:
```
{
    "creationDate": "2023-11-05T17:19:35.974556178Z",
    "id": "xxxxxxxxxxxx-xxxx-xxxxx-xxxxx-xxxxxxxxx",
    "key": "",
    "masterPasswordHash": "",
    "object": "auth-request",
    "origin": "https://my.own.domain",
    "publicKey": "xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx",
    "requestApproved": null,
    "requestDeviceType": "Chrome Browser",
    "requestIpAddress": "192.168.0.1",
    "responseDate": null
}
```

PUT status 422 payload:
```
{"key":"xxxxxxxxxxxxxxxxxxxxxxxxx","deviceIdentifier":"xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxx","requestApproved":true}
```
Response
```
{
  "error": {
    "code": 422,
    "reason": "Unprocessable Entity",
    "description": "The request was well-formed but was unable to be followed due to semantic errors."
  }
}
```

I am also a little confused as to why my requestIpAddress is my router's IP on my local network!? If I'm running from traefik shouldn't my IP be the public internet IP at the WAN?

End result is that I receive the notification but when I approve or deny nothing happens. On the cellphone I get an additional error: "We were unable to process your request. Please try again or contact us".

**Comment by BlackDex** [comment]
You are probably not running the `testing` tagged version as is mentioned in all other issues/discussions which report this.
Please check which version you are using and make sure it is `testing`, and also make sure you are using the new websocket proxy configuration which are provided at our wiki: https://github.com/dani-garcia/vaultwarden/wiki/Proxy-examples

**Comment by Majestic7979** [comment]
> You are probably not running the `testing` tagged version as is mentioned in all other issues/discussions which report this. Please check which version you are using and make sure it is `testing`, and also make sure you are using the new websocket proxy configuration which are provided at our wiki: https://github.com/dani-garcia/vaultwarden/wiki/Proxy-examples

Thanks, I don't think I was as clear as I could've been. I am NOT using 3012 websockets. I'm using the new Rocket based one. I can see the WSS in the console. I get instant sync between clients. This is the only issue I'm experiencing.

For reference this is how I spin up my container:

```
docker create --name=vaultwarden --restart=always -v /home/user1/dockerstore/vw-config/:/data/ -p 192.168.1.100:8088:80 vaultwarden/server:latest && docker start vaultwarden
```

So you're saying I have to spin up a container with a "testing" tag?

**Comment by BlackDex** [comment]
> So you're saying I have to spin up a container with a "testing" tag?

Yes, that is what i am saying, as is mentioned in all the other issues which reported this exact same error.
